src/Microsoft.Graph.Cli.Core/Microsoft.Graph.Cli.Core.csproj

```
@@ -44,6 +48,7 @@
     <PackageReference Include="Microsoft.Kiota.Http.HttpClientLibrary" Version="1.1.1" />
     <PackageReference Include="Spectre.Console" Version="0.47.0" />
     <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```

```suggestion
    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
```
Later version is available

---

Good catch :-)